smf/installinator/manifest.xml

```diff
@@ -4,16 +4,43 @@
 <service_bundle type='manifest' name='installinator'>
 
 <service name='system/illumos/installinator' type='service' version='1'>
-  <create_default_instance enabled='false' />
+  <create_default_instance enabled='true' />
   <single_instance />
 
-  <dependency name='multi_user' grouping='require_all' restart_on='none'
+  <!-- File systems should all be mounted before we begin: -->
+  <dependency name='fs-root' grouping='require_all' restart_on='none'
     type='service'>
-  <service_fmri value='svc:/milestone/multi-user:default' />
+    <service_fmri value='svc:/system/filesystem/root' />
+  </dependency>
+  <dependency name='fs-minimal' grouping='require_all' restart_on='none'
+    type='service'>
+    <service_fmri value='svc:/system/filesystem/minimal' />
+  </dependency>
+
+  <!-- The bootstrapping portion of the agent uses the network to communicate with
+    other servers on the rack. -->
+  <dependency name='loopback' grouping='require_all' restart_on='none'
+    type='service'>
+  <service_fmri value='svc:/milestone/network' />
+  </dependency>
```

Comment:
We wait on `svc:/milestone/network` here, but `svc:/milestone/network:default` below - is that intentional?

Reply:
No idea! This is a direct copy-paste from sled-agent. I'm not sure what this is doing.

Reply:
After chatting offline: cleaned up in 86540d2. Both installinator and sled-agent are cut down to just one `svc:/milestone/network` dependency (named `network` and does not include the `:default` suffix).